nsdra/.htaccess

```
@@ -2,12 +2,14 @@ Header set Access-Control-Allow-Origin *
 Options +FollowSymLinks
 RewriteEngine On
 
-RewriteRule ^$ https://www.nsdra.org/ [R=302,L]
+RewriteRule ^$ https://nsdra.org/ [R=302,L]
 
 RewriteRule "^maturity-indicator/readme/(.+)$" https://github.com/NSDRA/nsdra-maturity-indicators/blob/main/$1 [R=302,L]
 
 RewriteRule "^maturity-indicator/vocab/(.+)$" https://nsdra.org/vocab/$1 [R=302,L]
 
 RewriteRule "^assessment/(.+)$" https://nsdra.org/assessment/$1 [R=302,L]
 
-RewriteRule "^metadata-generator/$" https://nsdra.org/metadata-generator [R=302,L]
\ No newline at end of file
+RewriteRule "^metadata-generator/$" https://nsdra.org/metadata-generator [R=302,L]
+
+RewriteRule "^metadata-generator$" https://nsdra.org/metadata-generator [R=302,L]
```

You could simplify this with an optional `/`:
```suggestion
RewriteRule "^metadata-generator/?$" https://nsdra.org/metadata-generator [R=302,L]
```

---

Are you interested in making this change or should I just merge it as is?